LogoSvg.js

```diff
@@ -1,8 +1,13 @@
 `;
 
-export default () => <SvgCss xml={xml} width="100%" height="100%" />;
\ No newline at end of file
+export default function LogoSvg(props) {
+  return (
+    <SvgCss xml={xml} width="100%" height="100%" />
+  );
+};
```

Comment:
Changed syntax/format.

Reply:
In this PR it's hard to tell what has changed, it just looks like you created this file. 
In a different circumstance I'd want to know *why* you changed syntax and format. We can tell from the diff that this syntax/format has changed, but not why it has changed

Reply:
I had originally changed the syntax so that I could wrap the Svg with other tags within the component return statement so that I could directly apply styling to the container wrapping the Svg. Ultimately, it did not affect the styling I was looking for, but decided to leave the new syntax as is because it conformed with the return statements in all the other components of the app. Continuity.